src/modules/cargo.rs

```diff
@@ -8,6 +8,10 @@ use crate::data::PACKAGES_LIST;
 use crate::generators::gen_package_version;
 use crate::io::{csleep, dprint};
 
+pub fn get_signature() -> &'static str {
+    &"cargo run";
```

Comment:
All of these signatures should be just in the format:

```
     "cargo run"
```

So no `return`, no `&` and no `;`.